### PR TITLE
Acceptance test running on Docker container

### PR DESCRIPTION
**Pull request by PavelVesnin**

### CODE REVIEW

File: lib/prm/web/router.ex [hunk 1]
```diff
@@ -12,7 +12,6 @@ defmodule PRM.Web.Router do
   scope "/api", PRM.Web do
     pipe_through :api # Use the default browser stack
 
-    get "/", PageController, :index
```
Comment (AndrewDryga) [inline lib/prm/web/router.ex]: Should not be part of this PR.

File: config/config.exs [hunk 2]
```diff
@@ -10,6 +10,16 @@ config :prm,
...
 
+# Configure your database
+config :prm, PRM.Repo,
+  adapter: Ecto.Adapters.Postgres,
+  username: "postgres",
+  password: "postgres",
+  database: "prm",
+  hostname: "localhost",
+  pool_size: 10
+
```
Comment (AndrewDryga) [inline config/config.exs]: Should not be part of this PR.

File: bin/release.sh [hunk 3]
```diff
@@ -76,7 +76,7 @@ else
     exit 1
   else
     echo "[I] Creating git tag '${PROJECT_VERSION}'.."
-    echo "    Release Notes: "
+    echo "[I] Release Notes: "
```
Comment (AndrewDryga) [inline bin/release.sh]: This identation was for purpose, since this is two parts of one message.

File: .travis.yml [hunk 4]
```diff
@@ -56,7 +56,7 @@ script:
   - RUNNING_CONTAINERS=`docker ps | wc -l`;
   - if [ "${RUNNING_CONTAINERS//[[:space:]]/}" == "1" ]; then echo "[E] Container is not started\!"; docker logs prm --details --since 5h; exit 1; fi;
   # Run acceptance tests on Docker container
-  - "MIX_TEST_HOST=localhost MIX_TEST_PORT=4000 mix test test/acceptance"
+  - MIX_TEST_HOST=localhost MIX_TEST_PORT=4000 mix test test/acceptance
```
Comment (AndrewDryga) [inline .travis.yml]: Should not be part of this PR.

Comment (PavelVesnin) [inline .travis.yml]: this is Acceptance test running command. That's what pull request was created for

**Comment by gmile** [comment]
If I understand correctly, the goal is:

1. pack app into a container,
2. start the container (on certain host/port + ensuring it is connected to a DB located on the host environment),
2. poke some app's function from outside of container (e.g. host environment),
3. see that poking returned some expected value.

From what I understand when looking at the travis command execution sequence, this line:

```
MIX_TEST_HOST=localhost MIX_TEST_PORT=4000 mix test test/acceptance
```
will:

1. start the entire application on the host system,
2. run the test,
3. based on the config value, visit container's endpoint,
4. make an assertion on the value returned from container's endpoint.

IMO, If we're only trying to verify that container has been built successfully and is operational, a small sanity check should be enough, e.g. a script that does this:

1. a POST request that added something to DB,
2. [optional] a GET request to verify that written stuff is indeed returned from within container.

We can do that will much smaller effort – via a bash script. Here's an example: https://github.com/Nebo15/os.gateway/blob/master/bin/ci/sanity_test.sh

**Comment by PavelVesnin** [comment]
@gmile your understanding is correct. This is an acceptance tests that makes requests via HttpPoison on builded Docker container, so we can sure, that application was released correctly and works as expected.
